.github/workflows/script/relay_smoke_test.sh

```diff
@@ -0,0 +1,79 @@
+#!/bin/bash
+
+echo "Starting test network with a single node"
+
+./test/e2e/setup-e2e.sh
```

Comment:
I modified setup-e2e.sh, so it no longer runs solo network deploy, so you will need to add that step into your script after you merge/rebase the latest in main.